modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -381,7 +390,31 @@ definition {
 		task ("Then the MDF is submited and visible on listing page") {
 			WaitForPageLoad();
 
-			PRMMDFListing.viewOnList(campaignName = "Campaign Name");
+			PRMMDFListing.viewOnList(
+				campaignName = "Pending Marketing Review",
+				rowNumber = "Status");
+		}
+
+		task ("And when I change the MDF Request status") {
+			PRMMDFRequest.changeMDFStatus(
+				mdfName = "Company Name 1",
+				statusButton = "Approve");
+		}
+
+		task ("And when I go to the listing") {
+			PRMHomePage.gotoQuickLinksOption(
+				buttonLink = "/home",
+				buttonText = "Home");
+
+			PRMHomePage.gotoQuickLinksOption(
+				buttonLink = "mdf-requests",
+				buttonText = "MDF");
+		}
+
+		task ("Then the status of a MDF Request was change") {
+			PRMMDFListing.viewOnList(
+				campaignName = "Approved",
+				rowNumber = "Status");
```

Comment:
Hi @mateussandes23, here you are changing an existing test. You should add a new one.